### PR TITLE
Added comments detailing the use of lines 98 - 102

### PR DESCRIPTION
**Pull request by HansenML**
I added comments detailing why we need lines 98 - 102. On each cycle they place information into index which is then accessed by req.body.<someVarName> in the valStub.

### CODE REVIEW

File: test/routes/routerPost.js [hunk 1]
```diff
@@ -70,11 +71,14 @@ describe('in index/routes ', function () {
...
       });
     };
+    // Checks the local username, password, and teamName parameters
     let valStub = function (req) {
       return (req.body.username && req.body.password && req.body.teamName);
     };
 
+    // sets up our stubs for testing with authenticate and validate in the lib/auth module
     let stubs = { 'authenticate': authStub, 'validate': valStub };
+    // creates a proxy for calls to 'stubs' in the lib/auth module
```
Comment (chances) [inline test/routes/routerPost.js]: This is a proxy for authenticate and validate functions, not for stubs

Comment (chances) [inline test/routes/routerPost.js]: This comment is a bit redundant given the comment on line 79.

File: test/routes/routerPost.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ chai.use(dirtyChai);
 chai.use(chaiHttp);
 chai.use(chaiPromise);
 
+// This is a set of cases we wanted to test against for username, password, and teamName
```
Comment (chances) [inline test/routes/routerPost.js]: Move this comment to before line 16

**Comment by RickyV33** [comment]
I'm going to cancel this since you pulled from master!

**Pull request closed without merging**